### PR TITLE
[0.7] Astropy V2 Support

### PR DESCRIPTION
**Pull request by nabobalis**
A combination of items.

### CODE REVIEW

File: sunpy/coordinates/frameattributes.py [hunk 1]
```diff
@@ -3,17 +3,17 @@
 import datetime
 
 from astropy.time import Time
-from astropy.coordinates.baseframe import TimeFrameAttribute
+from astropy.coordinates import TimeAttribute
```
Comment (Cadair) [inline sunpy/coordinates/frameattributes.py]: this needs to be conditional, we need to keep support for older versions.

Comment (nabobalis) [inline sunpy/coordinates/frameattributes.py]: well what version do we want to support min? 1.3?

Comment (nabobalis) [inline sunpy/coordinates/frameattributes.py]: Because base frame wasn't needed for 1.3 versions in master at least.

Comment (Cadair) [inline sunpy/coordinates/frameattributes.py]: in 0.7 we support min 1.0.

In master we could bump to min 2.0

Comment (nabobalis) [inline sunpy/coordinates/frameattributes.py]: HOW ABOUT NOW?

**Pull request merged**